ext/vulnsrc/manual/manual.go

```diff
@@ -12,7 +16,1505 @@ type updater struct {
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         4.0,
+					"Score":               8.6,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:H/A:N",
+				},
+			},
+		},
+	},
+	// Debian unstable
+	{
+		Name: "CVE-2024-47175",
+		Namespace: database.Namespace{
+			Name:          "debian:unstable",
+			VersionFormat: dpkg.ParserName,
+		},
+		Description: "CUPS is a standards-based, open-source printing system, and `libppd` can be used for legacy PPD file support. The `libppd` function `ppdCreatePPDFromIPP2` does not sanitize IPP attributes when creating the PPD buffer. When used in combination with other functions such as `cfGetPrinterAttributes5`, can result in user controlled input and ultimately code execution via Foomatic. This vulnerability can be part of an exploit chain leading to remote code execution (RCE), as described in CVE-2024-47176.",
+		Link:        "https://security-tracker.debian.org/tracker/CVE-2024-47175",
+		// Debian did not assign a severity, so basing this on the score.
+		Severity: database.HighSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "cups",
+					Namespace: database.Namespace{
+						Name:          "debian:unstable",
+						VersionFormat: dpkg.ParserName,
+					},
+				},
+				Version: "2.4.10-2",
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-09-26T16:00Z",
+				"LastModifiedDateTime": "2024-09-30T16:00Z",
+				"CVSSv2": map[string]interface{}{
+					"ExploitabilityScore": 0.0,
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         4.0,
+					"Score":               8.6,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:N/I:H/A:N",
+				},
+			},
+		},
+	},
+
+	// Ubuntu 16.04
+	{
+		Name: "CVE-2024-47175",
+		Namespace: database.Namespace{
+			Name:          "ubuntu:16.04",
+			VersionFormat: dpkg.ParserName,
+		},
+		Description: "ppdCreatePPDFromIPP2 does not sanitize IPP attributes when creating the PPD buffer",
+		Link:        "https://ubuntu.com/security/CVE-2024-47175",
+		// Ubuntu gave this a severity lower than the severity derived from the CVSS score.
+		Severity: database.MediumSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "cups",
+					Namespace: database.Namespace{
+						Name:          "ubuntu:16.04",
+						VersionFormat: dpkg.ParserName,
+					},
+				},
+				Version: versionfmt.MaxVersion,
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-09-26T16:00Z",
+				"LastModifiedDateTime": "2024-09-27T16:00Z",
```

Comment:
nit: `1 October 2024` per the link (same for other ubuntus)

Reply:
I think they goofed. I'm going to just use last Thursday